sdk/servicebus/azure-servicebus/tests/utilities.py

```diff
@@ -9,19 +9,24 @@
 import time
 from azure.servicebus._common.utils import utc_now
 
-def get_logger(level):
-    azure_logger = logging.getLogger("azure")
-    if not azure_logger.handlers:
-        azure_logger.setLevel(level)
-        handler = logging.StreamHandler(stream=sys.stdout)
-        handler.setFormatter(logging.Formatter('%(asctime)s %(name)-12s %(levelname)-8s %(message)s'))
-        azure_logger.addHandler(handler)
-
-    uamqp_logger = logging.getLogger("uamqp")
-    if not uamqp_logger.handlers:
-        uamqp_logger.setLevel(logging.INFO)
-        uamqp_logger.addHandler(handler)
-    return azure_logger
+def _get_default_handler():
+    handler = logging.StreamHandler(stream=sys.stdout)
+    handler.setFormatter(logging.Formatter('%(asctime)s %(name)-12s %(levelname)-8s %(message)s'))
+    return handler
+
+def _build_logger(name, level):
```

Comment:
I would prefer logs not only being printed in console but also being persisted into files making it convenient for us to debug/read/search.

I have done something similar for EH, feel free to copy and adapt it into service bus:
https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/eventhub/azure-eventhub/stress/logger.py

Reply:
Truthfully, while I am familiar with the filehandlers, I was imagining for a V1 it would be effectively equivalent to adding a redirect on the console (`>`).  I can't argue too much against the formalization though, but would like to plumb the file target down properly in another parameter like I have the stress-test runtime.  Mind if I do that as a standalone issue to keep the changes encapsulated?

Reply:
(Created issue [here](https://github.com/Azure/azure-sdk-for-python/issues/14818))